libpod/oci_conmon_exec_linux.go

```diff
@@ -391,7 +391,11 @@ func (r *ConmonOCIRuntime) startExec(c *Container, sessionID string, options *Ex
 	}
 	defer processFile.Close()
 
-	args := r.sharedConmonArgs(c, sessionID, c.execBundlePath(sessionID), c.execPidPath(sessionID), c.execLogPath(sessionID), c.execExitFileDir(sessionID), ociLog, define.NoLogging, "")
+	tag := ""
+	if len(c.config.LogTag) > 0 {
```

Comment:
I don't think this check is necessary you can pass `c.config.LogTag` directly since if `len(c.config.LogTag) > 0` we are already passing `tag := ""` so I think you can pass `c.config.LogTag` directly to `sharedConmonArgs(`